src/cargo/core/features.rs

```diff
@@ -935,6 +997,67 @@ impl GitoxideFeatures {
             internal_use_git2: false,
         }
     }
+
+    fn expecting() -> String {
+        let fields = vec!["'all'", "'fetch'", "'checkout'", "'internal-use-git2'"];
+        format!(
+            "unstable 'gitoxide' only takes {} as valid inputs, your can use 'all' to turn out all gitoxide features, for shallow fetches see `shallow-index,shallow-deps`",
```

Comment:
could we use backtick "`" to quote options?

Reply:
Ok, it is a convention